tests/unit/unit1660.c

```diff
@@ -41,7 +41,7 @@ unit_stop(void)
 #if defined(CURL_DISABLE_HTTP) || defined(CURL_DISABLE_HSTS)
 UNITTEST_START
 {
-  return CURLE_OK; /* nothing to do when HTTP or HSTS are disabled */
+  /* return CURLE_OK;  nothing to do when HTTP or HSTS are disabled */
```

Comment:
A fix has been pendig for this here:
https://github.com/curl/curl/pull/14772/files#diff-aa6618147822be7a9ddb938893eaa6c9e8e7216a9e875147eb1bd1e5708143cd

Reply:
Let me know when it merge, and I will rebase.